Libraries/libcar/Headers/car/Rendition.h

```diff
@@ -44,13 +44,22 @@ class Rendition {
              * Raw data.
              */
             Data,
+            /*
+             * Extensions format
```

Comment:
I think this comment is unhelpful. What about `Raw data, WEBP format.` or deleting the comment entirely?

Reply:
Well, if I say "non-standard formats" would that be more helpful?